lib/cisco_node_utils/feature.rb

```diff
@@ -23,6 +23,25 @@ class Feature < NodeUtil
     # however, for test purposes it is sometimes convenient to support
     # feature disablement for cleanup purposes.
     # ---------------------------
+    def self.bfd_enable
+      return if bfd_enabled?
+      config_set('feature', 'bfd', state: '')
+    end
+
+    def self.bfd_disable
+      return unless bfd_enabled?
+      config_set('feature', 'bfd', state: 'no')
```

Comment:
In your email you indicated that bfd at times takes 20-30 seconds to disable.  We should keep and eye on that, especially for tests that disable bfd and then immediately try to re-enable it for the next test.  I would not add logic to handle that unless it's needed.